frontend/packages/knative-plugin/src/components/add/event-sources/EventSourceSection.tsx

```diff
@@ -16,17 +17,36 @@ import YAMLEditorSection from './YAMLEditorSection';
 import { EventSources } from '../import-types';
 import SinkSection from './SinkSection';
 import { isKnownEventSource } from '../../../utils/create-eventsources-utils';
+import { capabilityWidgetMap } from '@console/operator-lifecycle-manager/src/components/descriptors/spec/spec-descriptor-input';
 
 interface EventSourceSectionProps {
   namespace: string;
   fullWidth?: boolean;
   catalogFlow?: boolean;
+  kameletSource?: K8sResourceKind;
 }
 
+const getUISchema = (formSchema) => {
+  const uiSchema = {};
+  for (const k in formSchema.properties) {
+    if (formSchema.properties.hasOwnProperty(k)) {
```

Comment:
we don't need If block i think as we are iterating keys in properties itself

Reply:
This if the condition is required otherwise I am getting a lint error.